packages/datasource/src/form/tug/optionPreferredWayOfSharing.json

```diff
@@ -0,0 +1 @@
+["参与讨论技术问题", "回复他⼈技术困惑", "分享技术⼼得", "撰写技术⽂章", "线下演讲布道"]
```

Comment:
Will these texts be reused for any other pages? If not, I'd prefer to move them to `src/pages/tug/somewhere`...
Normally, we'd follow a feature-first folder structure. `packages/datasource` is only used for storing global values which need to be consumed by multiple pages or cross applications.